src/main/kotlin/com/chattriggers/ctjs/api/client/Sound.kt

```diff
@@ -523,9 +522,12 @@ class Sound(private val config: NativeObject) {
             throw NotImplementedError()
         }
 
-        override fun getName(): String {
-            return "CTJS_Sounds"
-        }
+        override fun getInfo() = ResourcePackInfo(
+            "chattriggers",
```

Comment:
Let's switch all the modids to `ctjs` (and make it a const on `CTJS` so we only have it in one place)